src/fit.jl

```diff
@@ -165,26 +184,65 @@ function eval_metrics!(results::Dict{String,Dict{String,Vector{Float64}}},
 end
 
 
-function store_scores!(results::Dict{String,Dict{String,Vector{Float64}}},
-                       estimator::LGBMEstimator, iter::Integer, evalname::String,
-                       scores::Vector{Cdouble}, metrics::Vector{String})
+function store_scores!(
```

Comment:
reformatted, and eating the new args (plus num_evals didnt need computing inside function as its being passed as an arg now)